python_transport/wirepas_gateway/transport_service.py

```diff
@@ -360,12 +360,27 @@ def on_data_received(
         self.mqtt_wrapper.publish(topic, event.payload, qos=1)
 
     def on_stack_started(self, name):
+        self.logger.debug("Sink started: %s", name)
         sink = self.sink_manager.get_sink(name)
         if sink is None:
             self.logger.error("Sink started %s error: unknown sink", name)
             return
 
         # Generate a setconfig answer with req_id of 0
+        self._generate_config_changed_answer(name)
+
+    def on_stack_stopped(self, name):
+        self.logger.debug("Sink stopped: %s", name)
+        sink = self.sink_manager.get_sink(name)
+        if sink is None:
+            self.logger.error("Sink stopped %s error: unknown sink", name)
+            return
+
+        # Generate a setconfig answer with req_id of 0
+        self._generate_config_changed_answer(name)
+
+    def _generate_config_changed_answer(self, name):
```

Comment:
We already have: _send_asynchronous_get_configs_response
So to keep code homogeneous, I would call this method:
_send_asynchronous_set_config_response